nextcord/threads.py

```diff
@@ -830,3 +830,17 @@ def _from_data(self, data: ThreadMemberPayload):
     def thread(self) -> Thread:
         """:class:`Thread`: The thread this member belongs to."""
         return self.parent
+
+    @property
+    def member(self) -> Optional[Member]:
+        """:class:`Member`: The member object of itself."""
+        return self.parent.guild.get_member(self.id)
+
+    async def get_member(self) -> Optional[Member]:
+        """:class:`Member`: Retrieves the :class:`Member` of itself.
```

Comment:
same here, there is better naming over
> of itself
such as, `` the guild :class:`Member` related to this member `` or similar, not exactly that